artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/Page.java

```diff
@@ -99,8 +91,6 @@ public Page(final SimpleString storeName,
       fileFactory = factory;
       this.storageManager = storageManager;
       this.storeName = storeName;
-      this.canBeMapped = fileFactory instanceof NIOSequentialFileFactory || fileFactory instanceof MappedSequentialFileFactory;
```

Comment:
For the original version, If we use NIOSequentialFileFactory, why we use mapped file read instead? Shouldn't it be consistent with the fileFactory type?

Reply:
In the original version it was a "cheap trick" to use NIO for write side, but mapped on read side